templates/config/openkm.properties

```diff
@@ -5,6 +5,7 @@ spring.datasource.username={{ openkm_db_user | mandatory }}
 spring.datasource.password={{ openkm_db_pass | mandatory }}
 
 # JPA stuff
+# TODO: fix for new installation
```

Comment:
I thought the check below for status from requests is the correct handling. Maybe we just are missing `ignore_errors`.